src/plugins/dashboard/public/application/components/dashboard_top_nav.tsx

```diff
@@ -97,6 +104,33 @@ const TopNav = ({
     setIsFullScreenMode(currentAppState?.fullScreenMode);
   }, [currentAppState, services]);
 
+  useEffect(() => {
+    const asyncSetIndexPattern = async () => {
+      if (dashboardContainer) {
+        let panelIndexPatterns: IndexPattern[] = [];
+        Object.values(dashboardContainer.getChildIds()).forEach((id) => {
```

Comment:
`dashboardContainer.getChildIds().forEach`?

Reply:
Yea, since that is already returning a list of children IDs, we should not need object.values